test/topology_experimental_raft/suite.yaml

```diff
@@ -13,6 +13,7 @@ run_first:
   - test_tablets
 skip_in_release:
   - test_raft_cluster_features
+  - test_crash_coordinator_before_streaming
```

Comment:
Use the new `skip_mode` marker instead (you can find examples in other tests, grep for them)

Reply:
used